sopel/plugins/handlers.py

```diff
@@ -128,47 +130,47 @@ def is_loaded(self):
         raise NotImplementedError
 
     def setup(self, bot):
-        """Setup the plugin with the ``bot``
+        """Setup the plugin with the ``bot``.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def has_setup(self):
-        """Tell if the plugin has a setup action
+        """Tell if the plugin has a setup action.
 
         :return: ``True`` if the plugin has a setup, ``False`` otherwise
         :rtype: bool
         """
         raise NotImplementedError
 
     def register(self, bot):
-        """Register the plugin with the ``bot``
+        """Register the plugin with the ``bot``.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def unregister(self, bot):
-        """Unregister the plugin from the ``bot``
+        """Unregister the plugin from the ``bot``.
 
         :param bot: instance of Sopel
         :type bot: :class:`sopel.bot.Sopel`
         """
         raise NotImplementedError
 
     def shutdown(self, bot):
-        """Take action on bot's shutdown
+        """Take action on bot's shutdown.
```

Comment:
```suggestion
        """Run the plugin's shutdown routine.
```

Similar to `setup()` above, this is only tied to the bot's shutdown because the bot first shuts down and unloads all plugins before shutting down itself.